### ISSUE TITLE
Can't get past event location from teleport event

### ISSUE THREAD

**Issue by NopeProsOnly** [SkriptLang/Skript#5392]
### Skript/Server Version

```markdown
[14:35:14 INFO]: [Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
[14:35:14 INFO]: [Skript] Skript's documentation can be found here: https://docs.skriptlang.org/
[14:35:14 INFO]: [Skript] Skript's tutorials can be found here: https://docs.skriptlang.org/tutorials
[14:35:14 INFO]: [Skript] Server Version: git-Paper-307 (MC: 1.19.2)
[14:35:14 INFO]: [Skript] Skript Version: 2.6.4
[14:35:14 INFO]: [Skript] Installed Skript Addons:
[14:35:14 INFO]: [Skript]  - skript-gui v1.3 (https://github.com/APickledWalrus/skript-gui)
[14:35:14 INFO]: [Skript]  - DiSky v4.8.0
[14:35:14 INFO]: [Skript]  - skript-fishing v1.2.2 (https://github.com/Fusezion/skript-fishing)
[14:35:14 INFO]: [Skript]  - skript-reflect v2.3 (https://github.com/TPGamesNL/skript-reflect)
[14:35:14 INFO]: [Skript]  - SkQuery v4.1.7
[14:35:14 INFO]: [Skript]  - SkBee v2.7.2 (https://github.com/ShaneBeee/SkBee)
[14:35:14 INFO]: [Skript] Installed dependencies:
[14:35:14 INFO]: [Skript]  - Vault v1.7.3-b131
[14:35:14 INFO]: [Skript]  - WorldGuard v7.0.8-beta-01+cbb2ba7
```

### Bug Description

I am unable to get the past event location

it says 

`There's no location in an on teleport event`

### Expected Behavior

it should support past event location

### Steps to Reproduce

```vb
on player teleporting:
  if player is in world "Lobby":
    if world of past event-location is in world "Survival" or world "Lobby_nether" or world "Lobby_the_end":
      set {SurvivalLoc::%player%} to world of past event-location
    if world of past event-location is in world "Creative":
      set {CreativeLoc::%player%} to world of past event-location
    if world of past event-location is in world "Oneblock" or world "Oneblock_Nether":
      set {OneblockLoc::%player%} to world of past event-location
  
  if world of past event-location is in world "Lobby":
    if player is in world "Survival":  
      wait a tick
      teleport player to {SurvivalLoc::%player%}
    if player is in world "Creative":
      wait a tick
      teleport player to {CreativeLoc::%player%}
    if player is in world "Oneblock":
      wait a tick
      teleport player to {OneblockLoc::%player%}
    if player is world "Bedwars":
      wait a tick
      make player execute command "/bw leave"
      clear player's inventory
```

I am using multiverse core btw

### Errors or Screenshots

![image](https://user-images.githubusercontent.com/78925970/214139634-c7a587de-e247-4f6a-a259-95bdd66db798.png)

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by Fusezion** [comment]
To clarify this isn't really a bug as much as just not supported. `PlayerTeleportEvent` the class used for this event doesn't have any location methods. In order to achieve location they'd have to use `EntityTeleportEvent` class.

---

## **PlayerTeleportEvent**
![image](https://user-images.githubusercontent.com/81952476/214143754-eaf33645-44fb-4fe6-960b-ebdb3bac53f7.png)

## **EntityTeleportEvent**
![image](https://user-images.githubusercontent.com/81952476/214143827-919c94a6-37d2-42f4-9d76-9c426616c9f2.png)

**Comment by colton-boi** [comment]
It does have location methods, It inherits them from PlayerMoveEvent 
![image](https://user-images.githubusercontent.com/71715967/214144402-b16aa8a2-64e5-4881-a95d-0b7ba198bb78.png)

**Comment by NopeProsOnly** [comment]
I know very little about java...
I do know this tho.
https://skripthub.net/docs/?id=1053
![image](https://user-images.githubusercontent.com/78925970/214144888-fb98e613-fb0a-45e0-bcba-2e9eecfb3969.png)

**Comment by Fusezion** [comment]
> It does have location methods, It inherits them from PlayerMoveEvent ![image](https://user-images.githubusercontent.com/71715967/214144402-b16aa8a2-64e5-4881-a95d-0b7ba198bb78.png)

alright let me rephrase that

> To clarify this isn't really a bug as much as just not supported. The team and original committer overlooked using this.

**Comment by colton-boi** [comment]
It's on the docs

**Comment by NopeProsOnly** [comment]
These docs? https://skripthub.net/docs/

**Comment by TheLimeGlass** [comment]
Pull request changing event location. https://github.com/SkriptLang/Skript/pull/4895

**Comment by TheLimeGlass** [comment]
Removing PR Available until that said pull request adds a fix for this.

**Comment by TheLimeGlass** [comment]
This also happens with `on portal`

**Comment by TheLimeGlass** [comment]
Appears https://github.com/SkriptLang/Skript/pull/4895 fixed this for `on portal`

**Comment by NopeProsOnly** [comment]
This issue is still open. So I'm assuming that this is still a issue. Has it been fixed in a pr? If so, then how can I download the pr. (Hopefully in jar version. I don't know how to compile.) I *can* wait for the next update release.

**Comment by TheLimeGlass** [comment]
@NopeProsOnly It's on schedule for 2.7.0-beta1 release. You can download the master branch currently by grabbing a nightly build from any of the ran actions https://github.com/SkriptLang/Skript/actions
